commitlint.config.js

```diff
@@ -13,7 +13,7 @@ module.exports = {
 };
 
 // Extra scopes supported outside the `modules` folder
-const scopes = ['labs'];
+const scopes = ['labs', 'pagination'];
```

Comment:
Remove this

Reply:
Possibly change the scoping script to pick up modules/packages within "_labs" instead